crates/interledger-stream/src/server.rs

```diff
@@ -85,37 +89,58 @@ impl ConnectionGenerator {
     }
 }
 
+#[derive(Serialize)]
+pub struct PaymentNotification {
+    pub to_username: Username,
+    pub from_username: Username,
+    pub destination: Address,
+    pub amount: u64,
+    pub timestamp: String,
+}
+
+/// A trait representing the Publish side of a pub/sub store
+pub trait PubStore {
```

Comment:
It would be better to make the name more specific to what it is. I would suggest something like:
```suggestion
pub trait StreamNotificationsStore {
```